### DISCUSSION TITLE
Assignment 1- Poker Problem (main)

### DISCUSSION THREAD

**Discussion by RishabhBajaj25** [chulminy/CIVE497-CIVE700#114] [Q&A]
My code passed all the tests individually, however, it fails when I paste the nine functions in the main problem. I can't seem to point out the error. Is anybody else facing the same problem?

**Comment by chulminy** [comment]
I think you need to capture error message to understand what your problems are.

**Reply by RishabhBajaj25** [reply]
![image](https://user-images.githubusercontent.com/70211310/105227915-e378f600-5b87-11eb-923f-d25c43cca473.png)
After copy and pasting the nine functions, this is the error I get after running the main problem.
